include/openenclave/trace.h

```diff
@@ -17,7 +17,9 @@ typedef void (*oe_log_callback_t)(
     oe_log_level_t level,
     uint64_t host_thread_id,
     const char* message);
+
 oe_result_t oe_log_set_callback(void* context, oe_log_callback_t callback);
+oe_result_t oe_set_host_log_level(oe_log_level_t log_level);
```

Comment:
nit: empty line between two declaration. Ideally, we would need to add the proper comments for every public APIs in this header.